docs/extending-documentation.md

```diff
@@ -134,19 +184,19 @@ module.exports = {
 };
 ```
 
-As you can see, we import the providence output file and pass it to the plugin options as the `changes` property.
+We import the providence output file (`providence-extend-docs-data.json'`) and pass it to the plugin options as the `changes` property.
 
-Now, the babel plugin will run for the files that we specify in `test` property, and replace the imports properly, as well as the tag names inside JavaScript code snippets!
+The babel plugin runs for the files that we specify in the `test` property, replaces the imports, and replaces the tag names inside JavaScript code snippets!
 
-> For the JavaScript code snippets, it only transforms the ones that use [MDJS](https://open-wc.org/mdjs/) syntax, e.g. \`\`\`js script, \`\`\`js story and \`\`\`js preview-story
+> It will only transforms JavaScript snippets that use [MDJS](https://open-wc.org/mdjs/) syntax such as \`\`\`js script, \`\`\`js story and \`\`\`js preview-story
```

Comment:
```suggestion
> It will only transform JavaScript snippets that use [MDJS](https://open-wc.org/mdjs/) syntax such as \`\`\`js script, \`\`\`js story and \`\`\`js preview-story
```